<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditZipCopyBackupDialog</class>
 <widget class="QDialog" name="EditZipCopyBackupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zip-And-Copy Backup Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="basicBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Basic properties</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="zipCopyBox">
     <property name="title">
      <string>Zip-and-copy properties</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Local Destination</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="localDestinationEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="closeButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditZipCopyBackupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditZipCopyBackupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
